packages/main/test/specs/TabContainer.spec.js

```diff
@@ -106,7 +106,7 @@ describe("TabContainer general interaction", () => {
 			}
 		});
 
-		assert.ok(tabHeight < tabScrollHeight, "Tab Content is scrollable");
+		assert.ok(tabHeight <= tabScrollHeight, "Tab Content is scrollable");
```

Comment:
Not sure about that. If tabHeight is equal to tabScrollHeight, then the tab content is not scrollable, but the assertion aims to confirm the opposite  - that is scrollable. I did not test it, but the spec is not correct anymore.